py3status/modules/scratchpad_async.py

```diff
@@ -48,17 +57,12 @@ def post_config_hook(self):
         t.start()
 
     def scratchpad_async(self):
-        response = {'cached_until': self.py3.CACHE_FOREVER}
-
-        if self.urgent:
-            response['urgent'] = True
-
-        if self.always_show or self.count > 0:
-            response['full_text'] = self.py3.safe_format(self.format, {'counter': self.count})
-        else:
-            response['full_text'] = ''
-
-        return response
+        return {
+            'urgent': self.urgent,
+            'cached_until': self.py3.CACHE_FOREVER,
+            'full_text': self.py3.safe_format(
+                self.format, {'counter': self.count}),
```

Comment:
Yes. Almost same! We need to find/make the correct formatter for more than 1. ;-)